src/tests/__snapshots__/babel.spec.js.snap

```diff
@@ -147,6 +194,35 @@ if (module.exports) {
 }"
 `;
 
+exports[`babel plugin transform should transform default named class 1`] = `
+"\\"use strict\\";
+
+Object.defineProperty(exports, \\"__esModule\\", {
+  value: true
+});
+exports.default = void 0;
+
+var _react = _interopRequireWildcard(require(\\"react\\"));
+
+function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) { var desc = Object.defineProperty && Object.getOwnPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : {}; if (desc.get || desc.set) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } } newObj.default = obj; return newObj; } }
+
+// eslint-disable-next-line
+class DefaultClass extends _react.Component {
+  render() {
+    return _react.default.createElement(\\"div\\", {
+      \\"data-51b7b868\\": true
+    }, this.props.children);
+  }
+
+}
+
+exports.default = DefaultClass;
+
+if (module.exports) {
+  module.exports.default.testId = \\"51b7b868\\";
```

Comment:
Maybe this one doesn't work well with `resolve` logic?
@lttb

Reply:
what do you mean? `resolve` logic does not depend on transpiled files